src/framework/preferences/css/stylus/Enactors.styl

```diff
@@ -25,9 +25,11 @@
     font-family: "Comic Sans MS", sans-serif !important;
 }
 
-// UI Enhancer "Enlarge text" link options
+// UI Enhancer "Enlarge text" input/button options
 .fl-text-larger input,
-.fl-text-larger button {
+.fl-text-larger button,
+.fl-text-larger .fl-input-enhance,
+.fl-text-larger .fl-button-enhance, {
```

Comment:
@cindyli i'm not sure what your question means. I'm not 100% sure what you're asking, but I'll try to answer. Yes, there will need to be docs, and this is only to provide them with a means of adding the enhancement when they aren't using a button or input element. Or was your question referring to the specific name for he class?